zipline/data/hdf5_daily_bars.py

```diff
@@ -726,21 +748,17 @@ def countries(self):
         return viewkeys(self._readers)
 
     def _country_code_for_assets(self, assets):
-        try:
-            country_codes = self._country_map.loc[assets]
-        except KeyError as exc:
-            raise_from(
-                NoDataForSid(
-                    'Assets not contained in daily pricing file: {}'.format(
-                        set(assets) - set(self._country_map)
-                    )
-                ),
-                exc
-            )
+        country_codes = self._country_map.get(assets)
 
-        unique_country_codes = country_codes.unique()
+        if country_codes is not None:
```

Comment:
When does pandas return None? I couldn't recreate that example in pandas 18 at least.

Reply:
It's happening in 22, which one of the travis builds uses.